subsys/mpsl/cx/software/mpsl_cx_software_rpc.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ *   This file implements Remote Procedure Calls (RPC) for Software Coexistence implementation.
+ *
+ */
+
+#include <mpsl/mpsl_cx_software.h>
+
+#include <nrf_rpc_cbor.h>
+#include <nrf_rpc/nrf_rpc_uart.h>
+
+#include <zephyr/device.h>
+
+/* TODO: unify the transport definition. */
+extern const struct nrf_rpc_tr rpc_tr;
+NRF_RPC_GROUP_DEFINE(mpsl_cx_rpc_group, "mpsl_cx", &rpc_tr, NULL, NULL, NULL);
+
+enum mpsl_cx_rpc_cmd {
+	MPSL_CX_RPC_CMD_SET_GRANTED_OPS = 0,
+};
+
+#ifdef CONFIG_MPSL_CX_SOFTWARE_RPC_CLIENT
+void mpsl_cx_software_set_granted_ops(mpsl_cx_op_map_t ops, k_timeout_t timeout)
+{
+	struct nrf_rpc_cbor_ctx ctx;
+	int64_t timeout_ms;
+
+	if (K_TIMEOUT_EQ(timeout, K_FOREVER)) {
+		timeout_ms = -1;
+	} else {
+		timeout_ms = k_ticks_to_ms_near64(timeout.ticks);
+	}
+
+	NRF_RPC_CBOR_ALLOC(&mpsl_cx_rpc_group, ctx, 11);
+	zcbor_uint_encode(ctx.zs, &ops, sizeof(ops));
+	zcbor_uint64_put(ctx.zs, timeout_ms);
```

Comment:
shouldn't be `zcbor_int64_put` ? `timeout_ms` is of `int64_t` type

Reply:
Good catch, thanks!